examples/webserver/streaming/src/main/java/io/helidon/examples/webserver/streaming/StreamingService.java

```diff
@@ -38,14 +38,13 @@
 public class StreamingService implements HttpService {
     private static final Logger LOGGER = Logger.getLogger(StreamingService.class.getName());
 
-    private final Path filePath;
+    // Last file uploaded (or default). Since we don't do any locking
+    // when operating on the file this example is not safe for concurrent requests.
+    private volatile File file;
 
     StreamingService() {
-        try {
-            filePath = Paths.get(Objects.requireNonNull(getClass().getResource(Main.LARGE_FILE_RESOURCE)).toURI());
-        } catch (URISyntaxException e) {
-            throw new RuntimeException(e);
-        }
+        // Default download file;
+        file = Paths.get(Main.LARGE_FILE_PATH).toFile();
```

Comment:
If we implement the suggestion above, we need the file to not exist.
Thus we can't use `File.createTemporaryFile`, instead we can resolve a file path in a temporary directory.

I.e. 

```java
try {
   file = Files.createTempDirectory("helidon-webserver-streaming-example").resolve("download.bin");
} catch (IOException e) {
    throw new UncheckedIOException(e);
}
```

Reply:
The code, as written, creates a new temp file every time.